configs/next/packages/binutils/sources

```diff
@@ -43,13 +43,10 @@ ATSRC_PACKAGE_BUNDLE=main_toolchain
 
 atsrc_get_patches ()
 {
-	at_get_patch \
-		https://raw.githubusercontent.com/powertechpreview/powertechpreview/master/GNU%20Binutils%20Patches/0001-Initialize-kept_size-in-gold-object.cc.patch \
-		4ab7c410728c6e4eeb9f44d6a295db57 || return ${?}
```

Comment:
This patch is no longer needed because it has been contributed upstream (see commit ID [f6f9ed015ea6](https://sourceware.org/git/?p=binutils-gdb.git;a=commitdiff;h=f6f9ed015ea6)).

OK.